tests/components/hassio/test_init.py

```diff
@@ -546,14 +547,14 @@ async def test_service_calls(
     )
     await hass.async_block_till_done()
 
-    assert aioclient_mock.call_count == 24
+    assert aioclient_mock.call_count == 22
```

Comment:
why the difference?

Reply:
Because now the test mocks the client library rather then the endpoint for addon info. This reduced the number of calls by 2.